src/dialog/enemyeditor.cpp

```diff
@@ -0,0 +1,1479 @@
+			ddl_attributes[0] = DropDownField(&local_guyref.misc1, list_enemies);
+			ddl_attributes[1] = DropDownField(&local_guyref.misc2, list_enemies);
+			ddl_attributes[2] = DropDownField(&local_guyref.misc3, list_enemies);
+			ddl_attributes[3] = DropDownField(&local_guyref.misc4, list_enemies);
+			ddl_attributes[9] = DropDownField(&local_guyref.misc10, list_digdoggermisc10);
+			break;
+		case eePATRA:
+			ddl_attributes[3] = DropDownField(&local_guyref.misc4, list_patramisc4);
+			ddl_attributes[4] = DropDownField(&local_guyref.misc5, list_patramisc5);
+			ddl_attributes[9] = DropDownField(&local_guyref.misc10, list_patramisc10);
+			ddl_attributes[19] = DropDownField(&local_guyref.misc20, list_patramisc20);
+			ddl_attributes[21] = DropDownField(&local_guyref.misc22, list_patramisc22);
+			ddl_attributes[24] = DropDownField(&local_guyref.misc25, list_patramisc25);
+			ddl_attributes[25] = DropDownField(&local_guyref.misc26, list_patramisc26);
+			ddl_attributes[28] = DropDownField(&local_guyref.misc28, list_patramisc28);
+			break;
+		case eePROJECTILE:
+			ddl_attributes[0] = DropDownField(&local_guyref.misc1, list_walkmisc1);
+			break;
+		case eeGHOMA:
+			ddl_attributes[0] = DropDownField(&local_guyref.misc1, list_gohmamisc1);
+			break;
+		case eeAQUA:
+			ddl_attributes[0] = DropDownField(&local_guyref.misc1, list_aquamisc1);
+			break;
+		case eeMANHAN:
+			ddl_attributes[1] = DropDownField(&local_guyref.misc2, list_manhandlamisc2);
+			break;
+		case eeLANM:
+			ddl_attributes[2] = DropDownField(&local_guyref.misc3, list_yesnomisc);
+			break;
+		case eeMOLD:
+			ddl_attributes[1] = DropDownField(&local_guyref.misc2, list_yesnomisc);
+			break;
+		case eeWIZZ:
+			ddl_attributes[0] = DropDownField(&local_guyref.misc1, list_wizzrobemisc1);
+			ddl_attributes[1] = DropDownField(&local_guyref.misc2, list_wizzrobemisc2);
+			ddl_attributes[3] = DropDownField(&local_guyref.misc4, list_yesnomisc);
+			break;
+		case eeDONGO:
+			ddl_attributes[9] = DropDownField(&local_guyref.misc10, list_dodongomisc10);
+			break;
+		case eeKEESE:
+			ddl_attributes[0] = DropDownField(&local_guyref.misc1, list_keesemisc1);
+			ddl_attributes[1] = DropDownField(&local_guyref.misc2, list_keesemisc2);
+			ddl_attributes[2] = DropDownField(&local_guyref.misc3, list_enemies);
+			break;
+		case eeLEV:
+			ddl_attributes[0] = DropDownField(&local_guyref.misc1, list_leevermisc1);
+			break;
+		case eeWALLM:
+			ddl_attributes[0] = DropDownField(&local_guyref.misc1, list_noyesmisc);
+			break;
+		case eeTRAP:
+			ddl_attributes[0] = DropDownField(&local_guyref.misc1, list_trapmisc1);
+			ddl_attributes[1] = DropDownField(&local_guyref.misc2, list_trapmisc2);
+			break;
+		case eeROCK:
+			ddl_attributes[9] = DropDownField(&local_guyref.misc10, list_rockmisc10);
+			break;
+		case eeNONE: //(Boss Death Triggers)
+			ddl_attributes[9] = DropDownField(&local_guyref.misc10, list_yesnomisc);
+			break;
+		default:
+			break;
+	}
+	// ATTRIBUTE SWITCHERS
+	
```

Comment:
(needs the misc refactor)
```suggestion
	auto attributes1_tab = Rows_Columns<3,8>();
	auto attributes2_tab = Rows_Columns<3,8>();
	auto attrib_tabs[] = {attributes1_tab,attributes2_tab};
	for(int q = 0; q < 32; ++q)
	{
		auto& tab = attrib_tabs[q/16];
		tab->add(l_attributes[q] = Label(fitParent = true, textAlign = 2));
		tab->add(ib_attributes[index] = Button(forceFitH = true, text = "?",
			disabled = true,
			onPressFunc = [&, index]()
			{
				InfoDialog("Attribute Info",h_attribute[index]).show();
			}));
		tab->add(sw_attributes[q] = Switcher(
			tf_attributes[q] = TextField(
				type = GUI::TextField::type::INT_DECIMAL,
				maxLength = 7,
				low = -999999,
				high = 999999,
				val = local_guyref.attributes[q],
				fitParent = true,
				onValChangedFunc = [&,q](GUI::TextField::type, std::string_view, int32_t val)
				{
					local_guyref.attributes[q] = val;
				}),
			ddl_attributes[q] = DropDownList(
				vPadding = 0_px,
				fitParent = true, selectedValue = local_guyref.attributes[q],
				onSelectFunc = [&,q](int32_t val)
				{
					local_guyref.attributes[q] = val;
				})
			));
	}
	
```